programs/management/commands/import_program_config.py

```
@@ -183,6 +189,99 @@ def handle(self, *args: Any, **options: Any) -> None:
+        )
+        if config_obj is None:
+            return None
+
+        data = config_obj.data
+        if isinstance(data, str):
+            try:
+                data = json.loads(data)
+            except json.JSONDecodeError as e:
+                raise CommandError(
+                    f"'counties_by_zipcode' config for white label '{white_label.code}' is not valid JSON: {e}"
+                )
```

Ruff (0.16.1)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)